config/locales/en/steps.yml

```diff
@@ -353,14 +356,25 @@ en:
         edit:
           page_title: Does your client have any Premium Bonds?
           heading:  Does your client have any Premium Bonds?
-
+      properties:
+        confirm_destroy:
+          page_title: Delete asset confirmation
+          heading: Are you sure you want to remove this asset?
+          confirm_delete_button: Yes, remove it
+          cancel_delete_button: No, do not remove it
+          confirm_delete_button_a11y: Yes, remove this asset
+          cancel_delete_button_a11y: No, do not remove this asset
+        destroy:
+          success_flash: The asset has been deleted
       residential_property:
```

Comment:
Would it also be possible to clean up this key name if we're reusing it across all asset types?